canonical/ubuntu.com
# Build /software-partnerships

## Done

- Build /software-partnerships based on [copydoc](https://docs.google.com/document/d/1ucKfDzLlw49sEM9OfkZ9aT5EHoDt4DsNebQ4ohzKn_E/edit) and [figma design](https://www.figma.com/file/F9BHRGM18jeaCfMr6BIGjT/23.10-U.com---Software-partnerships-(WD-4470)?node-id=58%3A1278&mode=dev)
- Deletes and removes references to 'model-driven-operations' as detailed at the top of the [copydoc](https://docs.google.com/document/d/1ucKfDzLlw49sEM9OfkZ9aT5EHoDt4DsNebQ4ohzKn_E/edit)
- Add software-partnerships to the footer where model-driven-operations used to be
- Add a redirect to for model-driven-operations to ubuntu.com/blog/tag/model-driven-operations

## QA

- Check the page matches the content of the copydoc and the figma desgin. Note: There are some small differences, Unless a difference in copy, go with the design.
- Check the redirect of /model-driven-operations works

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4473
